src/AppBundle/Entity/ProcurationRequest.php

```diff
@@ -273,6 +281,13 @@ class ProcurationRequest
     private $processedAt;
 
     /**
+     * @var bool
+     *
+     * @ORM\Column(type="boolean")
+     */
+    private $reminded = false;
```

Comment:
Instead of a boolean value, I would better like to see a counter. This way we can force to resend a reminder several times.